src/app/services/theme/theme.service.ts

```diff
@@ -102,6 +104,37 @@ export class ThemeService {
     }
   }
 
+  private overrideBootstrapColors(color: string) {
+    const variables: [string, number][] = [
+      ["--bs-link-color", 0],
+      ["--bs-link-hover-color", -0.1],
+      ["--bs-link-disabled-color", 0.1],
+    ];
+
+    const targetColor = hsl(color);
+
+    variables.forEach(([variable, lightnessAdjustment]) => {
+      const hslColor = targetColor.copy({ l: targetColor.l + lightnessAdjustment });
```

Comment:
double check but make sure the `l` value is clamped to `[0.0, 1.0]` - the d3 function is almost definitely doing this

Reply:
hsl lightness is clamped to [0,1] [d3 tests](https://github.com/d3/d3-color/blob/71c7f100f9fa85a1c70fcbaeb5f803ee8db5620d/test/hsl-test.js#L159-L164)